app/src/main/res/values/strings.xml

```diff
@@ -973,4 +973,8 @@
   <string name="overdue_search_no_search_results_title">No patients found</string>
   <string name="overdue_search_no_search_results_desc">You will see all overdue patients from your search here</string>
 
+  <!-- Selected overdue appointments -->
```

Comment:
Nit: Maybe we can rename the comment same as the epic name `<!-- Select and download overdue appointments -->`

Reply:
Changing this